db/migrate/20191105020642_create_flowchart_node.rb

```diff
@@ -7,10 +7,10 @@ def change
       t.string :header, null: false
       t.string :button_text
       t.string :next_question
-      t.references :child_id, foreign_key: { to_table: :flowchart_nodes }
```

Comment:
For next time, you can short hand these calls through the `belongs_to` associations.